Userland/Applications/KeyboardSettings/KeyboardSettingsWidget.cpp

```diff
@@ -107,9 +107,13 @@ class KeymapModel final : public GUI::Model {
     int row_count(GUI::ModelIndex const&) const override { return m_data.size(); }
     int column_count(GUI::ModelIndex const&) const override { return 1; }
 
-    GUI::Variant data(GUI::ModelIndex const& index, GUI::ModelRole) const override
+    GUI::Variant data(GUI::ModelIndex const& index, GUI::ModelRole role) const override
     {
-        return m_data.at(index.row());
+        String const& data = m_data.at(index.row());
+        if (role == GUI::ModelRole::BackgroundColor && data == m_active_keymap)
+            return Color(Color::Red);
```

Comment:
We try to avoid hard-coding colors, since the system is themeable.
Perhaps using a bold font for the active keymap could be a color-agnostic solution?

Reply:
Thanks for the review. I switched to using a bold default font. This looks a bit better.
In future PRs, I would like to add various UX improvements to the dialog. :)